code-samples/eventing/bookstore-sample-app/db/job.yaml

```diff
@@ -0,0 +1,24 @@
+apiVersion: batch/v1
+kind: Job
+metadata:
+  name: make-sample-table
+spec:
+  template:
+    spec:
+      containers:
+      - name: postgres-client
+        image: postgres:latest

```

Comment:
As stated in `deployment.yaml`, we are using the image from quay.io. I would suggest you to use the same to be consistent.